depends/cargo-checksum.sh

```diff
@@ -0,0 +1,13 @@
+echo "{\"files\":{`
+find . -type f |  	# Get list of file paths
+grep -v $1 |		# Exclude Makefile hashes
+grep -v .stamp_ |	# Exclude Makefile stamps
+sed 's|^./||' |		# Remove leading ./
+sort |			# Sort (for uniformity)
+xargs $2 |		# Get SHA256 hashes (assumes standard 'H(A) A' format)
+awk -v OFS='":"' '{print $2, $1}' |	#   'H(A) A'  ->  'A":"H(A)'
+sed 's|^|"|' |				#  'A":"H(A)' -> '"A":"H(A)'
+sed 's|$|"|' |				# '"A":"H(A)' -> '"A":"H(A)"'
+tr '\n' ',' |		# Concatenate lines with commas
```

Comment:
I think backslash-escaping within backticks is nonportable (POSIX allows the backslash sequence to be expanded early).

http://pubs.opengroup.org/onlinepubs/9699919799/xrat/V4_xcu_chap02.html#tag_23_02_06_03

Reply:
See also https://stackoverflow.com/questions/43838007/posix-shell-escaping-line-continuations-in-backquote-command-substitutions

Reply:
I used backticks originally because of the Makefile dollar-symbol issue. I'll change it now.